bin/virtme-ssh-proxy

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+import argparse
+import array
+import socket
+
+
+def get_hostname_cid(ssh_dst):
+    """Return the hostname and CID
+
+    >>> hostname, cid = get_hostname_cid("virtme-ng/24")
+    >>> hostname
+    'virtme-ng'
+    >>> cid
+    24
+    >>> hostname, cid = get_hostname_cid("virtme/ng/21")
+    >>> hostname
+    'virtme/ng'
+    >>> cid
+    21
+
+    """
+    splitted = ssh_dst.rsplit("/", 1)
+    hostname, cid = splitted[0], int(splitted[1], 10)
```

Comment:
that could cause a calltrace if the split didn't work (should always work), or if the CID is not an int. Maybe best to check if it is an int and print an readable error (or try to convert and print an error).

Reply:
Hmm I've implemented the try-and-catch-error approach.